xml/System/DateOnly.xml

```diff
@@ -939,7 +939,15 @@
         <summary>Converts the specified string representation of a date to its <see cref="T:System.DateOnly" /> equivalent using the specified format.
             The format of the string representation must match the specified format exactly or an exception is thrown.</summary>
         <returns>An object that is equivalent to the date contained in <paramref name="s" />, as specified by format.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+## Remarks
+ The <xref:System.DateOnly.ParseExact%28System.String%2CSystem.String%29?displayProperty=nameWithType> method parses the string representation of a date, which must be in the format defined by the `format` parameter. It also requires that the \<Date> element of the string representation of a date appear in the order specified by `format`, and that `s` have no white space other than that permitted by `format`.
+ The `format` parameter is a string that contains either a single standard format specifier, or one or more custom format specifiers that define the required format of `s`. For details about valid formatting codes, see [Standard Date and Time Format Strings](/dotnet/standard/base-types/standard-date-and-time-format-strings) or [Custom Date and Time Format Strings](/dotnet/standard/base-types/custom-date-and-time-format-strings).
+> [!NOTE]
+> If `format` is a custom format pattern that does not include date separators (such as "yyyyMMdd"), use the widest form of each custom format specifier. For example, if you want to specify months in the format pattern, specify the wider form, "MM", instead of the narrower form, "M".
```

Comment:
Not sure if this is still the best practice for DateOnly? Assuming it is.

Reply:
Yes, it is still same.